Formula/nss.rb

```diff
@@ -28,7 +29,6 @@ class Nss < Formula
   uses_from_macos "zlib"
 
   conflicts_with "resty", because: "both install `pp` binaries"
-  conflicts_with "googletest", because: "both install `libgtest.a`"
```

Comment:
You'll need to keep this for now to avoid the audit error. We can remove this line along with the corresponding one in `googletest` separately.